api/references/icons-in-labels.md

```diff
@@ -330,6 +330,7 @@ VS Code extensions can use these icons in labels, views, and trees.
 |<i class="codicon codicon-bold"></i>|bold|
 |<i class="codicon codicon-book"></i>|book|
 |<i class="codicon codicon-bookmark"></i>|bookmark|
+|<i class="codicon codicon-bracket-dot"></i>|bracket-dot|
```

Comment:
this codicon can be found here: https://github.com/microsoft/vscode/issues/134102#issuecomment-932422528, it's already on vscode codicon repository.